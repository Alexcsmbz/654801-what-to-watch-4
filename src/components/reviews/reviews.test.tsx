import Reviews from './reviews';

describe(`Reviews snapshot test`, () => {
  it(`Reviews should render movie details`, () => {
    const tree = renderer.create(
        <Reviews commentList={[]} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
