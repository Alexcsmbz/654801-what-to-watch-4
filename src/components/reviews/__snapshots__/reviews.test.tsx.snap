// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews snapshot test Reviews should render movie details 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <p
    style={
      Object {
        "color": "#0c0101",
      }
    }
  >
    No reviews yet...
  </p>
</div>
`;
